Makefile

```diff
@@ -37,20 +37,25 @@ SRC += _config.yml
 # Generate HTML files for publications
 .DELETE_ON_ERROR: _includes/%.html
 BIB_FILES := _includes/pubs.html _includes/patents.html
-_includes/%.html: bib/%.bib bib/publications.tmpl
+_includes/%.html: bib/pubbib/%.bib bib/pubbib/publications.tmpl
+	mkdir -p _includes
+	$(BIBBLE) $+ > $@
+
+SOF_BIB_FILES := _includes/software.html
```

Comment:
Can you consolidate Sof_bib_files and bib_files? Ideally under a new descriptive name.

Regardless both should be deleted by ‘make clean’